bio_hansel/main.py

```diff
@@ -263,30 +263,28 @@ def main():
 
     dfsummary = pd.DataFrame(subtype_results)
     dfsummary = dfsummary[SUBTYPE_SUMMARY_COLS]
+    dfsummary = dfsummary.dropna(axis=1, how='all')
 
     if output_summary_path:
+        dfsummary.to_csv(output_summary_path, sep='\t', index=None, float_format='%.3f')
```

Comment:
If you're feeling like you're repeating yourself a lot with the same key word arguments (`kwargs`) being used all over the place, you can just throw them into a dict:

```python
kwargs_for_pd_to_table = dict(sep='\t', index=None, float_format='%.3f')

# use kwargs...
df.to_csv(output, **kwargs_for_pd_to_table)
df2.to_csv(output2, **kwargs_for_pd_to_table)

```